vllm/model_executor/layers/logits_processor.py

```diff
@@ -76,13 +77,19 @@ def forward(
     def _get_logits(
         self,
         hidden_states: torch.Tensor,
-        lm_head: VocabParallelEmbedding,
+        lm_head: Union[VocabParallelEmbedding, LinearBase],
         embedding_bias: Optional[torch.Tensor],
     ) -> Optional[torch.Tensor]:
         # Get the logits for the next tokens.
-        logits = lm_head.linear_method.apply(lm_head,
-                                             hidden_states,
-                                             bias=embedding_bias)
+        linear_method = None
+        if isinstance(lm_head, LinearBase):
+            linear_method = lm_head.quant_method
```

Comment:
Is there a particular reason why this is called `quant_method` for `LinearBase` vs. `linear_method` for `VocabParallelEmbedding`. Just wondering if we could just rename it to `linear_method` in `LinearBase` and avoid this logic?